test/language/expressions/optional-chaining/call-expression-optional-chain.js

```diff
@@ -0,0 +1,17 @@
+/*---
```

Comment:
Can you add the copyright headers into each file? They are required to part tests correctly in different test runners used to consume Test262.